docs/assets/0123-import-scripts/README.md

```diff
@@ -0,0 +1,142 @@
+# Welcome to 0123-import-scripts!
+
+This is a template repository that can be used for the archiving process of a big dataset at the end of a research 
+project's lifetime. Download and unpack this repository from the 
+[excel2xml documentation page](https://docs.dasch.swiss/latest/DSP-TOOLS/dsp-tools-excel2xml/).  
+
+In this README, you will learn how to write a Python script for preparing data for an import into DSP.  
+
+Featuring: 
+
+- first steps with Visual Studio Code
+- the module `excel2xml` of dsp-tools
+- the benefits of Version Control with Git
+- the benefits of the Debugging Mode
+- extras: OpenRefine, Git GUIs, regexr
```

Comment:
This could probably be linked to the sections in the file (so you have like a clickable table of contents)?